bleak/backends/bluezdbus/manager.py

```diff
@@ -843,15 +844,21 @@ def _run_advertisement_callbacks(
             callback(device_path, device.copy())
 
 
+_instances: WeakKeyDictionary[Any, BlueZManager] = WeakKeyDictionary()
```

Comment:
How about calling this `_global_instances`?

Reply:
Also need to use `typing.Dict` for the type hint for compatibility with older Python versions.